src/components/utils/InputText.vue

```diff
@@ -68,6 +81,9 @@ export default Vue.component('input-text', {
         updateValue: function updateValue(newValue) {
             this.$emit('input', newValue);
         },
+        checkIfEdge() {
```

Comment:
This would be better as a computed property called `isEdgeBrowser` to be more clear